_data/events.yml

```diff
@@ -38,7 +38,8 @@
   url: https://local.hackcu.org
   
 - name: HackCU IV
-  location: TBD
+  date: 2018-02-24 8:00 AM
+  location: SEEC Building (East Campus)
   image-url: assets/img/hackcu.png
```

Comment:
Update image to something like this: 

https://beta.hackcu.org/assets/images/ogimage870x450.png